### DISCUSSION TITLE
How to edit user information (profile pic, username, etc...) with a database (ex. Prisma)

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [nextauthjs/next-auth#2520] [Help]
Hey i always wanted to know how to handle advanced user accounts pattern in a Next.js app. Does someone know how to edit user info. I use prisma as a database adaptater. 

Thanks for this amazing project !

**Comment by balazsorban44** [comment]
Hi, currently we don't have an API to edit users, you will have to do the changes yourself. There is a discussion to make this easier: https://github.com/nextauthjs/next-auth/issues/2269

For databases, I believe this should work: https://github.com/nextauthjs/next-auth/issues/2269#issuecomment-880626092
